### PR TITLE
add subtheme import conveniences

### PR DESCRIPTION
**Pull request by ianstarz**

### CODE REVIEW

File: lib/scss-preprocessor/broccoli-provide-theme-imports.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  var outDir = path.dirname(outFile);
+
+  mkdirp.sync(outDir);
+
+  var importPath = path.join(this.outputPath, this.parentMain).replace('.scss', '');
+  var relImportPath = path.relative(outDir, importPath);
+  var content = "@import '"+relImportPath+"';\n";
+
+  console.log(outFile);
+  console.log(content);
```
Comment (ianstarz) [inline lib/scss-preprocessor/broccoli-provide-theme-imports.js]: rm

**Comment by ianstarz** [comment]
pull in concert with
https://github.com/prototypal-io/ui-tomato-theme/pull/5
https://github.com/prototypal-io/ui-testapp/pull/3

File: lib/scss-preprocessor/index.js [hunk 2]
```diff
@@ -39,11 +41,26 @@ module.exports = function(tree, options) {
...
+    }
+
     return trees;
   }, []);
 
   var scssTree = funnel(tree, { include: ['**/*.scss'] });
   scssTree = mergeTrees([scssTree].concat(themeTrees), { overwrite: true });
 
+scssTree = require('broccoli-stew').log(scssTree, { output: 'tree', label: 'scss tree' });
+
```
Comment (ianstarz) [inline lib/scss-preprocessor/index.js]: rm :facepalm:

**Comment by HeroicEric** [comment]
👍

**Pull request merged**